.github/actions/on_device_tests/action.yaml

```diff
@@ -89,3 +92,61 @@ runs:
           --label author-${GITHUB_PR_HEAD_USER_LOGIN:-$GITHUB_COMMIT_AUTHOR_USERNAME} \
           --label author_id-${GITHUB_PR_HEAD_USER_ID:-$GITHUB_COMMIT_AUTHOR_EMAIL}
       shell: bash
+    - name: Download ${{ matrix.platform }} Test Results
+      if: always() && env.TEST_TYPE == 'unit_test'
+      env:
+        COBALT_XMLS_FILENAME: cobalt_xmls.zip
+        COBALT_ERROR_LOG: webDriverTestLog.ERROR
+        COBALT_INFO_LOG: webDriverTestLog.INFO
+        UNIT_TEST_RESULT_PATH: unit-test-results
+      run: |
+        set -uxe
+        i=0
+        while [ $i -lt 6 ]; do
```

Comment:
This is a bit opaque - what does 6 do here ? Just 6 retries ? A comment line would clarify

Reply:
I made it try 6 times with 10 seconds between tries => 60s total.

Added a comment.